cypress/support/step_definitions/anonymous_user_view_entity_HTML.js

```diff
@@ -10,22 +10,28 @@ when(/the anonymous user requests the entity specifying an Accept header with va
 		cy.get('@entityId').then((entityId) => {
 			const getEntityUrl = '/registry/' + registryName + '/entity/' + entityId;
 		
-			cy.request({
-				url: getEntityUrl,
-				headers: {
-					Accept: 'text/html'
-				}
-			}).then((response) => {
-				cy.wrap(response).as('htmlResponse')
-			})
+			cy.visit(getEntityUrl)
 		})
 	})
 })
 
 then('anonymous user can view the data in the given format', () => {
-	cy.get('@htmlResponse').then((response) => {
-		expect(response.body).to.have.string('<html>')
-		expect(response.body).to.have.string('<body>')
-		expect(response.body).to.have.string('<li data-automation-id="preferredLabel">')
+	cy.get('@registryName').then((registryName) => {
+		cy.get('@entityId').then((entityId) => {
+			const getEntityUrl = '/registry/' + registryName + '/entity/' + entityId;
+			cy.visit(getEntityUrl)
+
+			cy.get('li[data-automation-id=name]').contains('nameValue')

```

Comment:
I wonder if .contains('text') can have some effects where it matches 'subtext', rather than just 'text' as we expect? Not sure how this works, however and I don't see any immediate and clean assertion methods in the documentation